content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -37,8 +37,22 @@ by providing the following flags to the kube-apiserver:
 1. Create an egress configuration file such as `admin/konnectivity/egress-selector-configuration.yaml`.
 1. Set the `--egress-selector-config-file` flag of the API Server to the path of
 your API Server egress configuration file.
+1. If you use UDS connection, add volumes config to the kube-apiserver:
+```yaml
+spec:
+  containers:
+    volumeMounts:
+    - name: konnectivity-uds
+      mountPath: /etc/kubernetes/konnectivity-server
+      readOnly: false
+  volumes:
+  - name: konnectivity-uds
+    hostPath:
+      path: /etc/kubernetes/konnectivity-server
+      type: DirectoryOrCreate
+```
```

Comment:
pls indent line 41-53 by three spaces so the block aligns to the first "I" character on line 40.

Reply:
Done.